### PR TITLE
[Minor] Update CHANGELOG.md for 0.9

### PR DESCRIPTION
**Pull request by pedropaulosuzuki**
Some of the updates were in the wrong place (benchmark on features instead of Internal #428, cleanup of Math methods #514 is a internal refactor more than a bug fix).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Feature Enhancements:
   Implement global `Infinity` property (@AnirudhKonduru)
 - [FEATURE #410](https://github.com/boa-dev/boa/pull/410):
   Add support for the reviver function to JSON.parse (@abhijeetbhagat)
+- [FEATURE #525](https://github.com/boa-dev/boa/pull/425):
+  Implement additional `Math` methods (@mr-rodgers)
```
Comment (jasonwilliams) [inline CHANGELOG.md:16]: why did you change this to @mr-rodgers?
I don't see him on that PR

Comment (pedropaulosuzuki) [reply CHANGELOG.md:16]: Sorry, when I copy/pasted the [FEATURE #525], I forgot to update the link.

Didn't get the #514 part though, it is mentioned below in [Internal #514].

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ Feature Enhancements:
   Implement global `Infinity` property (@AnirudhKonduru)
 - [FEATURE #410](https://github.com/boa-dev/boa/pull/410):
   Add support for the reviver function to JSON.parse (@abhijeetbhagat)
+- [FEATURE #525](https://github.com/boa-dev/boa/pull/525):
+  Implement additional `Math` methods (@mr-rodgers)
```
Comment (HalidOdat) [inline CHANGELOG.md:16]:
```suggestion
```

`v0.9` has already been released, so this should be left for `v0.10`

Comment (pedropaulosuzuki) [reply CHANGELOG.md:16]: Hmm, ok!

File: CHANGELOG.md (hunk 1 above)
Comment (jasonwilliams) [reply CHANGELOG.md]: #525 looks like its still open so shouldn't really be on here.
`0.9.0` has been closed now, so any remaining PRs will go into `0.10.0`
